modules/images-using-images-s2i-python-pulling-images.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-using-images-s2i-python-pulling-images_{context}"]
+= Pulling images for Python
+
+These images come in two options:
+
+* RHEL 8
+* CentOS 7
+
+*RHEL 8 Based Images*
+
+The RHEL 8 images are available through the Red Hat Registry.
+
+.Procedure
+
+* To pull the RHEL 8 image, enter the following command for the version of Python you want:
++
+.Python `2.7`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/python-27-rhel7
+----
++
+.Python `3.3`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/openshift3/python-33-rhel7
+----
++
+.Python `3.4`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/python-34-rhel7
+----
++
+.Python `3.5`
+----
+$ podman pull registry.redhat.io/rhscl/python-35-rhel7
+----
+
+*CentOS 7 Based Images*
+
+These images are available on link:quay.io[Quay.io].
+
+.Procedure
+
+* To pull the CentOS 7 image, enter the following command for the version of Python you want:
++
+.Python `2.7`
+[source,terminal]
+----
+$ podman pull centos/python-27-centos7
+----
++
+.Python `3.3`
+[source,terminal]
+----
+$ podman pull openshift/python-33-centos7
+----
++
+.Python `3.4`
+[source,terminal]
+----
+$ podman pull centos/python-34-centos7
+----
++
+.Python `3.5`
+[source,terminal]
+----
+$ podman pull centos/python-35-centos7
```

Comment:
Above images need updated too,  registry.redhat.io/rhscl/python-27-rhel7:latest,  registry.redhat.io/ubi8/python-36:latest and registry.redhat.io/rhscl/python-38-rhel7:latest